### PR TITLE
Docs: Add ReturnToPrevious documentation

### PR DESCRIPTION
**Pull request by eledobleefe**
**What this PR does / why we need it**:
To share the `ReturnToPrevious` documentation.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by josmperez — CHANGES_REQUESTED** [review]
Hi, great work! I have done an editorial pass and would like to take a second look after you've reviwed the latest commit.

File: docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 id: add-return-to-previous-functionality
 title: Add 'Return to previous' functionality
-description: Offer an easy way of return to your context.
+description: Offer an easy way of return to allow users to return to their context.
```
Comment (eledobleefe) [inline docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md:4]: @josmperez wdyt of "Allow users to easily return to a specific context"?

**Review by L-M-K-B — APPROVED** [review]
Looks good from my side. I like a lot that you added the Usage guideline section.

File: docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md (hunk 1 above)
Comment (josmperez) [reply docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md:4]: works for me

File: docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+id: add-return-to-previous-functionality
+title: Add 'Return to previous' functionality
```
Comment (josmperez) [inline docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md:3]:
```suggestion
title: Allow users to easily return to a specific context
```

File: docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  - context
+---
+
+# Introduction
+
+A user's _context_ can switch dramatically while browsing Grafana, and sometimes their breadcrumbs don't match their history. In these cases, the users can't go back easily and may become frustrated with their experience. This guide defines _context_ and shows how you can add the `ReturnToPrevious` functionality to your plugin that solves this issue with minimal coding.
+
+## Context in Grafana
+
+For the purposes of Grafana plugin development, term _context_ refers to a user's place along a path from one root URL to another. Note that in Grafana, this term may also be used in connection with other Grafana features such as Explore or Dashboards.
```
Comment (josmperez) [inline docusaurus/docs/create-a-plugin/extend-a-plugin/add-return-to-previous-functionality.md:20]:
```suggestion
For the purposes of Grafana plugin development, the term _context_ refers to a user's place along a path from one root URL to another. Note that in Grafana, this term may also be used in connection with other Grafana features such as Explore or Dashboards.
```

**Review by josmperez — APPROVED** [review]
Ont nit, otherwise LGTM.

**Pull request merged**